scripts/utils/package/package.sh

```diff
@@ -0,0 +1,50 @@
+# shellcheck shell=sh
+# shellcheck disable=SC2039,SC2059
+
+# Title:          package
+# Description:    A library for package utils.
+# License-SPDX:   Apache
```

Comment:
Remove the license line since its the same as the rest of the build infrastructure. And Apache `2.0` license has SPDX `Apache-2.0` anyways.

https://github.com/termux/termux-packages/blob/master/LICENSE.md

https://spdx.dev/ids/